libs/shared/src/containers/Login/Login.stories.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Login as Component } from '.';
+import { withAccounts } from '@thecointech/storybookutils';
+import { AccountMap } from '../AccountMap';
+import { NormalizeAddress } from '@thecointech/utilities';
+
+export default {
+  title: "Shared/Login",
+  component: Component,
```

Comment:
This story doesn't quite work.  It shows the Login form, (so works for visual design) but the login button does not function :-(